org.gnome.Showtime.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "org.gnome.Showtime",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.llvm16"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin"
+  },
+  "command": "showtime",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures/Screenshots:create",
+    "--own-name=org.mpris.MediaPlayer2.org.gnome.Showtime"
```

Comment:
this name is actually granted by default. (based on the  app-id)

Reply:
I don't see what the doc properly update. A lot of app don't use the proper naming which is why they need to own the name.